From: =?utf-8?q?Guido_G=C3=BCnther?= <agx@sigxcpu.org>
Date: Mon, 4 Jan 2021 16:50:18 +0100
Subject: display-panel: Adjust to Librem 5 display size

This makes scaling work for the Librem 5's built-in display
---
 panels/display/cc-display-panel.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/panels/display/cc-display-panel.c b/panels/display/cc-display-panel.c
index dfca98c..915ef31 100644
--- a/panels/display/cc-display-panel.c
+++ b/panels/display/cc-display-panel.c
@@ -42,8 +42,8 @@
  * Note that WIDTH is assumed to be the larger size and we accept portrait
  * mode too effectively (in principle we should probably restrict the rotation
  * setting in that case). */
-#define MINIMUM_WIDTH 740
-#define MINIMUM_HEIGHT 530
+#define MINIMUM_WIDTH  720
+#define MINIMUM_HEIGHT 360
 
 #define PANEL_PADDING   32
 #define SECTION_PADDING 32
